harmony/harmonydb/sql/20241029-mk12-filters.sql

```diff
@@ -0,0 +1,129 @@
+-- This table allows creating custom pricing rules
+CREATE TABLE market_mk12_pricing_filters (
+    number BIGSERIAL PRIMARY KEY, -- Filter number
```

Comment:
Maybe instead of numbers do unique names with `[a-z0-9_-]`? Will look much more readable in the webui

Reply:
Done